packages/koa-shopify-auth/src/auth/create-enable-cookies.ts

```diff
@@ -0,0 +1,440 @@
+import {Context} from 'koa';
+
+import Error from './errors';
+import {OAuthStartOptions} from '../types';
+
+const HEADING = 'Enable cookies';
```

Comment:
Does quilt have a mechanism to support i18n?

Reply:
Seems like no: https://github.com/Shopify/quilt/pull/268#discussion_r215017108

I think it's worth asking the dev tools team to consider this as part of their backlog though!